rpmvenv/extensions/python/venv.py

```diff
@@ -109,6 +109,7 @@ def generate(config, spec):
         spec.blocks.files.append('/%{venv_install_dir}')
 
         spec.blocks.install.append('%{venv_cmd} %{venv_dir}')
+        spec.blocks.install.append('cd %{SOURCE0}')
```

Comment:
This seems like it could have a greater impact than only affecting symlink handling. Is it necessary for this change to work?